docs/reference/issuers/acme/dns01.rst

```diff
@@ -76,17 +76,17 @@ Cert-manager requires the following IAM policy.
            {
                "Effect": "Allow",
                "Action": "route53:GetChange",
-               "Resource": "arn:aws:route53:::change/*"
+               "Resource": "*"
            },
            {
                "Effect": "Allow",
                "Action": "route53:ChangeResourceRecordSets",
-               "Resource": "arn:aws:route53:::hostedzone/*"
+               "Resource": "*"
            },
            {
                "Effect": "Allow",
                "Action": "route53:ListHostedZonesByName",
-               "Resource": "arn:aws:route53:::hostedzone/*"
+               "Resource": "*"
```

Comment:
In my IAM rules, this is the only one I had to change to `*`

Reply:
this is right according to https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/r53-api-permissions-ref.html

Reply:
Agree with this. For my rules I had to do something similar and set

```
            "Effect": "Allow",
            "Action": [
                "route53:GetChange",
                "route53:ListHostedZonesByName"
            ],
            "Resource": "*"
```

and

```
            "Effect": "Allow",
            "Action": "route53:ChangeResourceRecordSets",
            "Resource": "arn:aws:route53:::hostedzone/<zone id>"
```